src/triage/cli.py

```diff
@@ -59,6 +60,81 @@ def stampdb(self, args):
               f"we think your results schema is version {revision} and are upgrading to it")
         stamp_db(revision, args.dbfile)
 
+
+@Triage.register
+class Experiment(Command):
+    """Validate and run experiments, manage experiment database"""
+
+    def __init__(self, parser):
+        parser.add_argument(
+            '-c', '--config',
+            type=argparse.FileType('r'),
+            help="config file for Experiment",
+            required=True
```

Comment:
dunno that it makes sense, but you could always default to `experiment.yaml`, as with `database.yaml`

Reply:
My reasoning against this is that projects tend to have multiple experiments, and it's preferable for them to not have a generic name like 'experiment'. Generally, the names should indicate something about what they are experimenting with.

Database creds are more static, and almost always will just have one per project.

Reply:
Ah hah. OK.

Maybe then, and especially considering it's required and not an "option," it should be a straight-forward argument:

    parser.add_argument('config', type=FileType('r'), help="...")

CLI:

    triage experiment --validate-only experiment7.yaml